Jenkinsfile

```diff
@@ -214,8 +215,10 @@ pipeline {
                     steps {
                       retry(3) {
                         container('maven') {
-                            sh '.ci/scripts/docs/prepare.sh'
-                            sh '.ci/scripts/docs/build.sh'
+                            configFileProvider([configFile(fileId: 'maven-nexus-settings-zeebe-local-repo', variable: 'MAVEN_SETTINGS_XML')]) {
+                                sh '.ci/scripts/docs/prepare.sh'
+                                sh '.ci/scripts/docs/build.sh'
+                            }
```

Comment:
I wasn't sure about this. It was a conflict because the docs are no longer build in newer versions.